xds/internal/client/bootstrap/bootstrap.go

```diff
@@ -185,7 +189,10 @@ func NewConfig() (*Config, error) {
 	}
 
 	if config.BalancerName == "" {
-		return nil, fmt.Errorf("xds: Required field %q not found in bootstrap", "xds_servers.server_uri")
+		return nil, fmt.Errorf("xds: Required field %q not found in bootstrap %s", "xds_servers.server_uri", jsonData["xds_servers"])
+	}
+	if config.Creds == nil {
```

Comment:
Optional: Move these checks to the end of `case "xds_servers":`.

Reply:
If moved, it won't fail if the file doesn't have "xds_servers".

Reply:
Right. Makes sense.